lib/loader.js

```diff
@@ -28,7 +32,8 @@ module.exports = function(content, map) {
 		to: loaderUtils.getCurrentRequest(this),
 		query: query,
 		minimize: this.minimize,
-		loaderContext: this
+		loaderContext: this,
+        sourceMap: sourceMap
```

Comment:
@evilebottnawi Feel free to convert tabs to spaces if you are in the mood, will be done via the upcoming `webpack-defaults` upgrade anyways :)